actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -82,13 +82,10 @@ def view_assigns
     # <tt>render :file => "foo/bar"</tt>.
     # :api: plugin
     def _normalize_args(action=nil, options={})
-      case action
-      when ActionController::Parameters
-        unless action.permitted?
-          raise ArgumentError, "render parameters are not permitted"
-        end
+      if action.respond_to?(:permitted?)
```

Comment:
Doesn't this change behavior? We are now raising if action respods to `permitted?` not *if* action is permitted or not?